internal/resources/policy/scope/constants.go

```diff
@@ -8,9 +8,9 @@ package scope
 const (
 	ManagementClusterNameKey = "management_cluster_name"
 	ProvisionerNameKey       = "provisioner_name"
-	ClusterNameKey           = "name"
+	ClusterNameKey           = "cluster_name"
```

Comment:
Since this resource has already been released, we should introduce new attributed and mark the old key as deprecated. 
Can you please do this changes to ensure backward compatibility